CMP1903_A1_2324/Game.cs

```diff
@@ -8,14 +10,48 @@ namespace CMP1903_A1_2324
 {
     internal class Game
     {
-        /*
-         * The Game class should create three die objects, roll them, sum and report the total of the three dice rolls.
-         *
-         * EXTRA: For extra requirements (these aren't required though), the dice rolls could be managed so that the
-         * rolls could be continous, and the totals and other statistics could be summarised for example.
-         */
+       public List<int> Roller()
+        {
+            int sum = 0;
+            List<int> returnList = new List<int>(); // a list created which holds the values that will be returned from this method
+            List<string> possibleAnswers = new List<string>() { "yes","no" }; // holds values of the two expected answers from the user when asked if they wish to continue
+            string answer = "yes";
+            while (answer != "no") // loops until user enters "no" after being asked if they wish to roll again
+            {
 
-        //Methods
+                Die die1 = new Die(); // creating new die objects
+                Die die2 = new Die();
+                Die die3 = new Die();
+                int firstRoll = die1.Roll(); // calling the objects to run their roll method which return an int value
+                int secondRoll = die2.Roll();
+                int thirdRoll = die3.Roll();
+                sum = (firstRoll + secondRoll + thirdRoll); // creating a sum of the three returned int values for each dies face value
```

Comment:
This section could be done in a while or for loop and have a running total? This would also give you another chance to show object encapsulation!